src/cmd/bootstrap.ts

```diff
@@ -95,25 +139,32 @@ export const bootstrapValues = async (): Promise<void> => {
     ['core.yaml', 'docker-compose.yml'].map((val) => copyFile(`${rootDir}/${val}`, `${env.ENV_DIR}/${val}`)),
   )
 
-  let originalValues
+  // FIXME: should be it the last step of bootstraping values?
+  await genSops()
+  if (existsSync(`${env.ENV_DIR}/.sops.yaml`) && existsSync(`${env.ENV_DIR}/.secrets`)) {
+    await encrypt()
+    await decrypt()
+  }
+  let originalValues: Record<string, any> = {}
+  let generatedSecrets
   if (isChart) {
-    // write chart values if we got any
-    originalValues = getChartValues()
+    originalValues = getInputValues() as Record<string, any>
```

Comment:
The getChartValues is misleading to me because it makes me think about values stored in charts/ directory.
Do you have any other suggestion?

Reply:
Also what is nice, if you have a single values file, and point the env variable to it. Then it can also be used from cli.

When you want to quickly share a config, as a single yaml for example?

Reply:
ok, cli point taken...I am always using that input file now for cli as well indeed actually...